src/main/java/de/mediathekview/mlib/filmlisten/FilmlisteLesen.java

```diff
@@ -404,7 +404,7 @@ public void progress(final long bytesRead, final long size) {
 
   private InputStream selectDecompressor(final String source, InputStream in) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)

Reply:
No refactoring. So no Change here.